besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -692,12 +692,12 @@ void setBannedNodeIds(final List<String> values) {
   private String metricsPrometheusJob = "besu-client";
 
   @Option(
-      names = {"--host-whitelist"},
+      names = {"--host-allowlist", "--host-whitelist"},
```

Comment:
should we deprecate the --host-whitelist option? It may be confusing not to state that the new allowlist is the prefered one.